vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -129,8 +139,22 @@ XamlView NativeUIManager::reactPeerOrContainerFrom(xaml::FrameworkElement fe) {
   return nullptr;
 }
 
+// Previous versions of react-native-windows did not configure Yoga to use "LegacyStretchBehaviour"
+// We now set it, but to allow apps a period to transition, we are adding a property that can be set to
+// maintain the previous behavior.
+winrt::Microsoft::ReactNative::IReactPropertyName LegacyStretchBehaviourProperty() noexcept {
+  static winrt::Microsoft::ReactNative::IReactPropertyName propName =
+      winrt::Microsoft::ReactNative::ReactPropertyBagHelper::GetName(
+          winrt::Microsoft::ReactNative::ReactPropertyBagHelper::GetNamespace(L"ReactNative.NativeUIManager"),
+          L"LegacyStretchBehaviour");
+  return propName;
+}
+
 NativeUIManager::NativeUIManager(Mso::React::IReactContext *reactContext) {
   m_context = reactContext;
+  m_useLegacyStretchBehaviour =
+      winrt::unbox_value_or<bool>(m_context->Properties().Get(LegacyStretchBehaviourProperty()), true);
```

Comment:
>true [](start = 97, length = 4)

This seems to me that this is in conflict with the description of the pr. My interpretation was that the default is new behavior and the legacy behavior is opt-in. Therefore I would have expected to see `false` here.